app/main.py

```diff
@@ -1,3 +1,42 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
```

Comment:
The implementation is not correct and tests should be failing locally. Make sure that flake8 and pytest are passing locally and send the task again